zipline/assets/_assets.pyx

```diff
@@ -203,6 +203,26 @@ cdef class Equity(Asset):
         params = ', '.join(strings)
         return 'Equity(%d, %s)' % (self.sid, params)
 
+    property security_start_date:
+        """
+        DEPRECATION: This property should be deprecated and is only present for
```

Comment:
Yea, you can use [warnings.warn] (https://docs.python.org/2/library/warnings.html#warnings.warn), passing DeprecationWarning.  I think we have examples of this elsewhere in the code.